warehouse/templates/index.html

```diff
@@ -13,7 +13,7 @@
 -#}
 {% extends "base.html" %}
 
-{% block title %}{% if testPyPI %}Test{% endif %}PyPI – the Python Package Index{% endblock %}
+{% block title %}{% if testPyPI %}Testing – {% endif %}The Python Package Index{% endblock %}
```

Comment:
Not sure about this one. I'm wondering if it would be better if all titles (not just ones that fall back on this default) start with "PyPI -" or "TestPyPI -"

Reply:
Currently we have:
- PyPI – the Python Package Index · PyPI
- TestPyPI – the Python Package Index · TestPyPI

I think this is repetitive...

I'll update to:

- PyPI · The Python Package Index
- TestPyPI · The Python Package Index